sites/firehouse.com/server/styles/index.scss

```diff
@@ -41,3 +41,45 @@ $body-bg: #fff;
     }
   }
 }
+
+// Override to make card deck flows with 4 cols display 25% till it reaches a lower resolution
+@media (min-width: 768px) {
+  .card-deck-flow {
+    $self: &;
+    &--4-cols {
+      #{ $self }{
+        &__node {
+          flex: 0 0 25%;
```

Comment:
I believe you can use the media breakpoint and col Mixins to handle this

Reply:
See https://getbootstrap.com/docs/4.4/layout/overview/

And the bottom of this page:

https://getbootstrap.com/docs/4.4/layout/grid/

Reply:
If those don’t work let me know and I’ll merge as-is

Reply:
Worked Great!